examples/tutorial/00_Setup.ipynb

```diff
@@ -13,67 +13,58 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This first step to the tutorial will make sure your system is set up to do all the remaining sections, with all software installed and all data downloaded as needed.  The [index](index.ipynb) provided some links you might want to examine before you start."
+    "This first step to the tutorial will ensure your system is set up to handle all the subsequent sections, with all software installed and all data downloaded as needed. The [index](index.ipynb) provides some links you might want to examine before you start."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Getting set up\n",
-    "\n",
-    "Please consult [holoviz.org](http://holoviz.org/installation.html) for the full instructions on installing the software used in these tutorials. Here is the condensed version of those instructions, assuming you have already downloaded and installed [Anaconda](https://www.anaconda.com/download) or [Miniconda](https://conda.io/miniconda.html) and have opened a command prompt in a Conda environment:"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "```\n",
-    "conda create -n project \"anaconda-project>=0.11\"\n",
-    "conda activate project\n",
-    "anaconda-project download pyviz/holoviz_tutorial\n",
-    "cd holoviz_tutorial # You may need to delete this directory if you've run the command above before\n",
-    "anaconda-project run jupyter notebook\n",
+    "```{important} \n",
+    "The instructions below are specific to this tutorial. If you are looking to install a HoloViz package for your own use, please consult each library's website (linked from [HoloViz](https://holoviz.org/)) for detailed guidance. Typically, installing these packages simply involves `conda install` or `pip install`.\n",
```

Comment:
```suggestion
    "The instructions below are specific to this tutorial, and are designed to ensure that you install exactly the same versions for which the tutorial was written so that the output shown will exactly match what you see yourself. If you are looking to install a HoloViz package for your own use, please consult each library's website (linked from [HoloViz](https://holoviz.org/)) for detailed guidance. Typically, installing these packages simply involves `conda install` or `pip install`, e.g. `conda install hvplot datashader`.\n",
```